docs/reference.rst

```diff
@@ -621,7 +621,7 @@ Settings Schema Definition
     Optional keys: ``kwargs``
 
 
-- ``transport_cache_time_in_seconds`` - ``integer``: If enabled, uses a per-service transport cache that is keyed off the service name and transport settings, persists across all clients in memory, and expires after this number of seconds. By default, a new transport is created for every new client. Note that this is safe in a multi-processing environment, but not in a multi-threaded environment. You should disable this in a multi-threaded environment. (additional information: ``{u'gte': 0}``)
+- ``transport_cache_time_in_seconds`` - ``anything``: This field is deprecated. The transport cache is no longer supported. This settings field will remain in place for a month or so to give a safe period for people to remove it from settings, but its value will always be ignored.
```

Comment:
Whoever is gonna read that in the future, won't know `a month or so` since when (w/o having to look into git lot)

Reply:
Good point! I have reworded this with an exact date.